helpers/object_utils.ts

```diff
@@ -32,3 +32,10 @@ export function isNil<T>(obj: T | null | undefined): obj is null | undefined {
 export function deepCopy<T>(obj: T): T {
   return JSON.parse(JSON.stringify(obj));
 }
+
+export function isPromise<T>(obj: any): obj is Promise<T> {
+  // This method should generally be avoided but can be used for high throughput / performance
```

Comment:
If we're trying to match internal coda comment styling, this advice to callers of the function would be best as a jsdoc /**...*/ comment above line 36.

Reply:
yeah, can do that. was just copying this method over from coda repo. I'll probably tweak the language a bit before moving them to the jsdoc comment.